docker-images/pysyft-worker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-slim
+
+ENV WORKER_DIR /worker
+
+# Install PySyft
+RUN apt-get update && apt-get install -y gcc
```

Comment:
Consider adding `&& rm -r /var/lib/apt/lists/*`
To cleanup apt files not needed in the image

Reply:
As far as I know, deleting those files will not reduce the image size in anyway and only delete those files in the last layer but keeping them in previous ones

Reply:
It won't keep files if you add `rm` in the same `RUN` command where apt-get is executed, e.g.:
`RUN apt-get update && apt-get install -y gcc && rm -rf /var/lib/apt/lists/*`

Reply:
I think this will make the directive too long and harder to read, the packages are now only 17mb, I will switch to this if it gets way bigger than this